boxes.yaml.example

```diff
@@ -44,6 +44,7 @@ centos7-bats-ci:
       bats_tests:
         - "swapfile.bats"
         - "fb-install-katello.bats"
```

Comment:
One of the biggest things we need is https://github.com/evgeni/forklift/blob/99fdda8d44631373e75ece1bb3aefdc2a970f52a/bats/fb-install-katello.bats#L47-L52 which was put in to help out on the CI system as far as I remember.

So maybe we need like a setup role to do a few of those items (instead of "testing" them). Otherwise I think it's safe to remove the install test file.